cgp/hl_api.py

```diff
@@ -44,7 +45,12 @@ def evolve(
         Number of parallel processes to be used. If greater than 1,
         parallel evaluation of the objective is supported. Currently
         not implemented. Defaults to 1.
-
+    reorder_genome: bool, optional
+        Whether genome reordering should be applied.
+        If True, reorder is applied to the parents (genome) before creating offspring.
+        Since the parents genome changes to a reordered genome at every generation,
+        this creates a neutral drift through the search space.
```

Comment:
```suggestion
        If True, reorder is applied to the parents genomes at every generation before creating offspring. Reorder randomizes the genotype of an individual without changing the phenotype.
```

Reply:
i remove the "neutral drift" part not bc it's wrong, but i think it's not really helpful here. maybe this should be added somewhere in the documentation, maybe of the population class?

Reply:
Added a comment in `Population.reorder_genome`